crates/sp-domains-fraud-proof/src/storage_proof.rs

```diff
@@ -58,6 +60,7 @@ pub enum FraudProofStorageKeyRequest {
     BlockDigest,
     RuntimeRegistry(RuntimeId),
     DynamicCostOfStorage,
+    DomainSudoCall(DomainId),
```

Comment:
Similar to the new variant in the `StatelessDomainRuntimeCall`, even the `DomainSudoCall` won't be constructed until the consensus runtime upgrade but this essentially updates the `FraudProofStorageKeyRequest` (argument of the runtime API) I'm not sure if it is compatible.

Reply:
Same as above. This is the last index so Runtime should not be aware of this. Scale encoding shouldn't be issue AFAIK